integration_tests/test_suites/auto_materialize_perf_tests/test_asset_daemon_perf.py

```diff
@@ -89,7 +89,10 @@ def test_auto_materialize_perf(scenario: PerfScenario):
         start = time.time()
 
         AutomationConditionEvaluator(
-            entity_keys=AssetSelection.all().resolve(asset_graph),
+            automation_condition_by_key={
+                key: check.not_none(asset_graph.get(key).automation_condition)
+                for key in asset_graph.all_asset_keys
+            },
             instance=instance,
             asset_graph=asset_graph,
```

Comment:
I don't see why you need to pass in `automation_condition_by_key`. Can you just access the asset graph directly from within the evaluator?

Reply:
yeah was trying to get a bit fancy in an effort to avoid the `check.not_none(asset_graph.get().automation_condition())` inside of `evaluate_asset`.

Not worth in hindsight, updated